package/package_esp32_index.template.json

```diff
@@ -523,57 +523,57 @@
           ]
         },
         {
-          "version": "3.0.0-gnu12-dc7f933",
+          "version": "4.0.2-db0513a",
           "name": "mklittlefs",
           "systems": [
             {
-              "host": "aarch64-linux-gnu",
-              "url": "https://github.com/earlephilhower/esp-quick-toolchain/releases/download/3.0.0-gnu12/aarch64-linux-gnu.mklittlefs-c41e51a.200706.tar.gz",
-              "archiveFileName": "aarch64-linux-gnu.mklittlefs-c41e51a.200706.tar.gz",
-              "checksum": "SHA-256:fc56e389383749e4cf4fab0fcf75cc0ebc41e59383caf6c2eff1c3d9794af200",
-              "size": "44651"
+              "host": "aarch64-apple-darwin",
+              "url": https://github.com/earlephilhower/mklittlefs/releases/download/4.0.2/aarch64-apple-darwin-mklittlefs-db0513a.tar.gz",
```

Comment:
```suggestion
              "url": "https://github.com/earlephilhower/mklittlefs/releases/download/4.0.2/aarch64-apple-darwin-mklittlefs-db0513a.tar.gz",
```

Reply:
Sorry, don't have a computer to do real git checkin.  Will fix later this week.

Reply:
I can fix it if you don't mind.

Reply:
Fixed